libs/ml/include/ml/saveparams/saveable_params.hpp

```diff
@@ -37,8 +37,10 @@ struct OpsSaveableParams
 
   virtual ~OpsSaveableParams() = default;
 
-  fetch::ml::OpType op_type     = OpType::NONE;
-  bool              is_training = true;
+  fetch::ml::OpType                    op_type     = OpType::NONE;
+  bool                                 is_training = true;
+  std::vector<fetch::math::SizeVector> batch_input_shapes{};
```

Comment:
if you're adding these two parameters to saveable params then you need to update the serialisers affected by this right?

Reply:
oh, I've missed it, sure!